docs/ldml/tr35-keyboards.md

```diff
@@ -950,9 +956,9 @@ where a flick to the Northeast then South produces two code points.
 ### <a name="Element_import" href="#Element_import">Element: import</a>
 
 The `import` element is used to reference another xml file so that elements are imported from
-another file. The use case is to be able to import a standard set of vkeys, transforms, and similar
-from the CLDR repository.  `<import>` is not recommended as a way for keyboard authors to
-split up their keyboard into multiple files, as the intent is for each single XML file to contain all that is needed for a keyboard layout.
+another file. The use case is to be able to import a standard set of `vkey`s, `transform`s, and similar
```

Comment:
Noting that `vkey` has been deleted in another recent PR.

Reply:
yes, already fixed and merged in that PR. i'll be merging that into this one shortly…